app/src/main/java/se/inera/intyg/minaintyg/logging/ErrorController.java

```diff
@@ -0,0 +1,32 @@
+package se.inera.intyg.minaintyg.logging;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import se.inera.intyg.minaintyg.logging.dto.LogErrorRequestDTO;
+import se.inera.intyg.minaintyg.logging.service.LogErrorService;
+import se.inera.intyg.minaintyg.logging.service.dto.LogErrorRequest;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/error")
+public class ErrorController {
```

Comment:
I commented on one of @ChrilleSigerhed comments, but I think it got lost :-)

I believe it should be called LogController to say what it does. Then it makes very much sense to call the endpoint /error. So /api/log/error.

Reply:
Fixed